problems/2704_toBe_or_notToBe/compare-functions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param {string} val
+ * @return {Object}
+ */
+
+function expectFunc (expectedVal) {
+    return {
+        toBe: function (actualVal) {
+            if (expectedVal !== actualVal) throw new SyntaxError('Not Equal');
```

Comment:
зависит от кодстайла в команде, но я предпочитаю всегда оборачивать в блок


```suggestion
            if (expectedVal !== actualVal) {
              throw new SyntaxError('Not Equal');
            }
```

прибавилось две строки, но код стал консистентнее, что улучшает скорость чтения кода, т.к. зачастую встречается код именно в синтаксисие `if () { ... }`, чем `if () ...`